Tools/build/mypy.ini

```
@@ -1,5 +1,5 @@
 [mypy]
-files = Tools/build/generate_sbom.py
+files = Tools/build/generate_sbom.py, Tools/build/compute-changes.py
```

Can we make this a multi-line list? Ideally we could use a TOML file, does MyPy support `mypy.toml` or `.mypy.toml`?

---

It supports `pyproject.toml`, but it can affect stuff. + Our CI expects all folders to have `mypy.ini`. Current version of `mypy` does not support multiline strings here. But, I will fix that upstream. So, we can expect this in 1.16.0 somewhere.

---

Seems to work like this? 
```suggestion
files =
    Tools/build/compute-changes.py,
    Tools/build/generate_sbom.py
```
(Unfortunately can't add a trailing comma to the last entry.)

---

I've created a MyPy issue to request support for `mypy.toml` or similar: https://github.com/python/mypy/issues/18617

---

I will fix it shortly, sorry for the `,` bug 😢 
I've fixed it several times already, but it is very persistent for some reason.